test-suite/coqdoc/bug5700.tex.out

```diff
@@ -20,14 +20,14 @@
 
 \begin{coqdoccode}
 \end{coqdoccode}
-\begin{verbatim}foo (* bar *) \end{verbatim}
- \begin{coqdoccode}
+\texttt{ foo (* \{bar\_bar\} *) } 
```

Comment:
I found `\texttt` wrong too but `\verb` would need to invent in advance (*) a symbol which is not present in the text to show verbatim. So, I stayed with `\texttt`, knowing that `_`, `\`, etc are already escaped.

(*) with a less imperative `coqdoc`, this would be easy, but I don't see how to do it here easily.